mono/metadata/icall-internals.h

```diff
@@ -65,18 +62,6 @@ mono_icall_get_logical_drives (void);
 
 guint32
 mono_icall_drive_info_get_drive_type (MonoString *root_path_name);
-
-MonoBoolean
-mono_icall_get_process_working_set_size (gpointer handle, gsize *min, gsize *max);
-
-MonoBoolean
-mono_icall_set_process_working_set_size (gpointer handle, gsize min, gsize max);
-
-gint32
-mono_icall_get_priority_class (gpointer handle);
-
-MonoBoolean
-mono_icall_set_priority_class (gpointer handle, gint32 priorityClass);
```

Comment:
If the implementation moves from icall -> process please align changes with uwp source files accordingly since it have implementation in icall-windows-uwp.c that should be moved to process-windows-uwp.c for affected methods.